homeassistant/components/romy/entity.py

```diff
@@ -0,0 +1,24 @@
+"""Base entity for ROMY."""
+
+from romy import RomyRobot
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class RomyEntity(Entity):
```

Comment:
```suggestion
class RomyEntity(CoordinatorEntity[RomyVacuumCoordinator], Entity):
```

Reply:
Can move the CoordinatorEntity to the base class

Reply:
moved